server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -105,6 +105,21 @@ public String getTextBody() {
         return textBody;
     }
 
+    public boolean isActiveAtDate(ZonedDateTime zonedDateTime) {
+        Preconditions.checkNotNull(zonedDateTime);
+        return isEnabled
+            && isAfterOrEqualToFromDate(zonedDateTime)
+            && isBeforeOrEqualToToDate(zonedDateTime);
+    }
+
+    private Boolean isAfterOrEqualToFromDate(ZonedDateTime zonedDateTime) {
+        return fromDate.map(date -> date.isBefore(zonedDateTime) || date.equals(zonedDateTime)).orElse(true);
+    }
+
+    private Boolean isBeforeOrEqualToToDate(ZonedDateTime zonedDateTime) {
```

Comment:
is before or equal to ToDate which makes in snake case isBeforeOrEqualToToDate

Sorry no funny mistaken jokes about toto ^^